app/lib/utils/userFriendlyErrors.js

```diff
@@ -0,0 +1,15 @@
+const userFriendlyErrors = {
+  'Error: 11 OUT_OF_RANGE: EOF':
+    "The person you're trying to connect to isn't available or rejected the connection.\
+ Their public key may have changed or the server may no longer be responding."
+}
+
+const errorToUserFriendly = error => {
```

Comment:
this func can be a 1 liner and much simpler:

`const errorToUserFriendly = error => userFriendlyErrors[error] || error`